### DISCUSSION TITLE
Version 1.6.2 linux deb version. Cannot cancel offers

### DISCUSSION THREAD

**Discussion by naotodatena** [Chia-Network/chia-blockchain#14275] [General]
Generated offers in 1.6.1 or generated offers with shell: it is impossible to cancel.

A lot of messages but the important one is " no sufficient mojos"

**Comment by blupino** [comment]
I think this is same issue with 

https://github.com/Chia-Network/chia-blockchain/issues/14232

**Reply by naotodatena** [reply]
Yes same issue

**Comment by hajes** [comment]
you have to cancel without sending fee/payment to blockchain. there is some weird bug that reserves 50 xch for 6 xch offer
